.github/actions/report-job-status/action.yml

```diff
@@ -0,0 +1,41 @@
+name: 'Send VK Teams notification on failure'
+description: 'Send message to VK Teams channel/bot on failure'
+
+inputs:
+  job-steps:
+    required: true
+    description: Steps context in JSON
+  api-url:
+    required: true
+    description: VK Teams API URL
+  bot-token:
+    required: true
+    description: VK Teams bot token
+  chat-id:
+    required: true
+    description: VK Teams main chat ID
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Set chat for reports
+      run: |
+        # Report failures on commits in stable branches to the common chat;
+        # report failures in PRs to individual chats
+        if ${{ github.ref == 'refs/heads/master' ||
```

Comment:
Try this condition to cover all branches:

```
if [[ "${{ github.ref }}" == "refs/heads/master" || 
      "${{ github.ref }}" =~ refs/heads/[0-9].[0-9]+$ ]]; then
  ...
else
  ...
fi
```

Reply:
Done